app/gui/language/ast/macros/src/token.rs

```diff
@@ -28,7 +26,7 @@ pub fn spaceless_ast(input: proc_macro::TokenStream) -> proc_macro::TokenStream
 /// Inner logic for `derive_has_tokens`.
 pub fn derive_for_enum(decl: &syn::DeriveInput, data: &syn::DataEnum) -> TokenStream {
     let ident = &decl.ident;
-    let params = decl.generics.params.iter().collect_vec();
+    let params = decl.generics.params.iter().collect::<Vec<_>>();
```

Comment:
why this change, how does it affect the time?

Reply:
`collect_vec` was part of trait imported from prelude.